rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
+
+Start a REST-AT coordinator on port 8080
+
+    cd <narayana-repo>/jboss-as/build/target/wildfly-28.0.0.Final
+    cp docs/examples/configs/standalone-rts.xml standalone/configuration
+    ./bin/standalone.sh -c standalone-rts.xml
+
+VERIFY THAT IT IS RUNNING, eg:
+
+    curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+BUILD  AND RUN THE DEMO APPLICATION on port 8081
+
+    cd <narayana-repo>/rts/sra
+    mvn clean package -Dquarkus.package.type=uber-jar
+    java $JAVA_OPTS -Dquarkus.http.port=8081 -jar target/sra-participant-runner.jar &
+    
+    OR
+    mvn quarkus:dev
+
+Book a trip within a transaction (see sra/demo/api/TripController.java method bookTrip)
+this starts a transaction and you can verify it is running using
+curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+    curl -XPOST http://localhost:8081/trip/book?hotelName=Rex
+
+EXPECTED OUTPUT
+---------------
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating hotel participant state to: TransactionPrepared
+
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating trip participant state to: TransactionPrepared
+
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating trip participant state to: TransactionCommitted
+
+    SRA: 0_ffff0a4cf0b6_-568c278f_6458bef8_7fb: Updating hotel participant state to: TransactionCommitted
+
+
+EXAMPLE OF HOW TO MANUALLY TEST THAT THE COORDINATOR IS WORKING USING curl
+---------------------------------------------------------------------------
+
+    curl -H "Content-Type: application/x-www-form-urlencoded" -X POST http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+    curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+    curl -X PUT --data txstatus=TransactionCommitted http://localhost:8080/rest-at-coordinator/tx/transaction-manager/0_ffff0a4cf0b6_-568c278f_6458bef8_7fb/terminator
+    curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+WHAT JUST HAPPENED?
+-------------------
+
+1. We started REST-AT coordinator which is available to use in wildfly.
+
+2. We verified REST-AT coordinator by invoking the transaction manager url.
+
+3. Build and run the sar application, it starts the service on same port i.e 8081.
+
+4. Invoke book trip which start SRA and calls the hotel and flight service.
+
+5. The example performed an HTTP POST request and check that the transaction status code is 200.
+
+6. The example performs commit operation if the status code is 200.
```

Comment:
What is issuing the commit operation?
What sequence of actions caused it to be issued?

Reply:
> What is issuing the commit operation? What sequence of actions caused it to be issued?

@mmusgrov Please review this, if it is not per expected then could you please provide me the best one.
Thanks.